### PR TITLE
refactor: refactor job task history page and changed API spec

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### Things to Talk About
plz check by commits

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue [hunk 1]
```diff
@@ -269,11 +268,12 @@ const getJobs = async () => {
 
 /* Watcher */
 watch(() => state.selectedStatus, (selectedStatus) => {
+    console.log(selectedStatus);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue:271]: nitpick: 🧐

File: apps/web/src/services/asset-inventory/collector/collector-history/collect-job/CollectJobPage.vue [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
-        <job-task-details v-if="selectedItem"
-                          :selected-item="selectedItem"
+        <job-task-details v-if="state.selectedItem"
+                          :selected-item="state.selectedItem"
         />
     </div>
 </template>
 
-<script lang="ts">
+<script setup lang="ts">
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-history/collect-job/CollectJobPage.vue:27]: praise: 👍👍👍

**Review by yuda110 — APPROVED** [review]
LGTM! I checked mainly based on conventions.

**Review by sulmoJ — APPROVED** [review]
Thank you for your hard refactor!

**Pull request merged**